### PR TITLE
Remove flag CONFIG and creation of KfDef from options

### PR DESCRIPTION
**Pull request by zhenghuiwang**
Th flag about setting config file or url should use the FILE (--file) option instead of CONFIG (--config) option.

This PR removes CONFIG.

Related to: #4239

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4282)

### CODE REVIEW

File: bootstrap/pkg/kfapp/coordinator/coordinator.go [hunk 1]
```diff
@@ -254,7 +254,7 @@ func CreateKfDefFromOptions(options map[string]interface{}) (*kfdefsv3.KfDef, er
 	}
 
 	// If a config file is specified, construct the KfDef entirely from that.
-	configFile := options[string(kftypesv3.CONFIG)].(string)
+	configFile := options[string(kftypesv3.FILE)].(string)
```
Comment (jlewi) [inline bootstrap/pkg/kfapp/coordinator/coordinator.go]: Can we just delete the function CreateKfDefFromOptions? I don't think it is used anymore.
This function was used for backwards compatibility in 0.6 to specify all the different command line flags and then create a KfDef from it.

Comment (zhenghuiwang) [reply bootstrap/pkg/kfapp/coordinator/coordinator.go]: I deleted all functions that relies on options.

**Comment by zhenghuiwang** [comment]
/test kubeflow-presubmit


```
util.py                     71 INFO     time="2019-10-14T22:32:03Z" level=error msg="Creating kfctl-5aba error: &{Code:RESOURCE_ERROR Location:/deployments/kfctl-5aba/resources/kfctl-5
aba Message:{\"ResourceType\":\"gcp-types/container-v1beta1:projects.locations.clusters\",\"ResourceErrorCode\":\"FAILED_OPERATION\",\"ResourceErrorMessage\":\"The network \\\"default\
\\" does not have available private IP space in 10.0.0.0/8 to reserve a /14 block for containers for cluster {us-central1-a, 29647740582, kubeflow-ci-deployment, kfctl-5aba, 86644c1580
f31e54842f322f4a076d95465b1244822bb2a895d58c0dd1d543c6}. - https://container.googleapis.com/v1beta1/projects/29647740582/zones/us-central1-a/clusters/kfctl-5aba\"} ForceSendFields:[] N
ullFields:[]}" filename="gcp/gcp.go:380"
```

**Pull request merged**